tools/cxbe/Main.cpp

```diff
@@ -113,10 +113,7 @@ int main(int argc, char *argv[])
             }
             else if(strcmp(szOptionU, "TITLE") == 0)
             {
-                if(dwParamSize > 40)
```

Comment:
Looks like this variable no longer needed after this change.
I will remove it; I'll leave a message and resolve this conversation when I've done that.

Reply:
Fixed